parser/testdata/config_err_use_from_other_service.txt

```diff
@@ -0,0 +1,38 @@
+# Verify calls to config.Load are called
+! parse
+stderr 'A config instance can only be referenced from within the service that the call to'
+
+-- svc/svc.go --
+package svc
+
+import (
+    "context"
+
+    "encore.dev/config"
+)
+
+type Config struct {
+    FooEnabled bool
+}
+
+var Cfg = config.Load[Config]()
+
+
+//encore:api
+func Subscriber1(ctx context.Context) error {
+    return nil
+}
+
+
+-- libraries/foo/foo.go --
+package foo
+
+import (
+    "test/svc"
+)
+
+func init() {
+    if svc.Cfg.FooEnabled {
```

Comment:
We need to support accepting the config objects as inputs to functions inside libraries. Is that still supported?

Reply:
Yes, because the resource usage tracker only tracks the _original_ identifier the configuration instance is loaded onto.

As such, you can reference it and pass the whole instance to a library function and that will work fine - as long as that reference to it is done from within the original service that the config is loaded in

Reply:
Ok great!